tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -107,10 +108,19 @@ public void AcquireTokenTest()
         {
             TestCommon.ResetInternalStaticCaches();
 
-            _mobileTestHelper.AcquireTokenTestHelper(
-                _xamarinController,
-                LabUserHelper.GetDefaultUserAsync().GetAwaiter().GetResult(),
-                CoreUiTestConstants.AcquireTokenInteractive);
+            try
+            {
+                _mobileTestHelper.AcquireTokenTestHelper(
+                    _xamarinController,
+                    LabUserHelper.GetDefaultUserAsync().GetAwaiter().GetResult(),
+                    CoreUiTestConstants.AcquireTokenInteractive);
+            }
+            catch (FileLoadException ex)
+            {
+#pragma warning disable CA2201 // Do not raise reserved exception types
+                throw new Exception(ex.FusionLog);
```

Comment:
By doing this you lose the original exception details, which may be important. All .net exceptions have a concept of wrapping other exceptionsd, which you can use here. I would rewrite this as: 

```csharp
throw new FileLoadException($"A FileLoad exception happened. This usually occurs when a dependency DLL is not found (e.g. Newtonsoft.Json.dll). See inner exception for details. Fusion Log: {ex.FusionLog}. ", ex);

```

Reply:
There changes are only for testing appcenter stuff, will be removed later. please ignore

Reply:
There's not reason to not leave this code in permanently.